Kernel/Arch/aarch64/CPUID.cpp

```diff
@@ -0,0 +1,509 @@
+/*
+ * Copyright (c) 2023, Konrad <konrad@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/Arch/aarch64/CPUID.h>
+
+namespace Kernel {
+
+// https://developer.arm.com/downloads/-/exploration-tools/feature-names-for-a-profile
+StringView cpu_feature_to_name(CPUFeature::Type const& feature)
```

Comment:
Could you merge this commit (Kernel: Add Aarch64 CPUFeature enumeration name mapping ) with the previous one?

Reply:
sure!